s3stream/src/main/java/com/automq/stream/s3/operator/AbstractObjectStorage.java

```diff
@@ -353,6 +355,11 @@ public CompletableFuture<Void> delete(List<ObjectPath> objectPaths) {
         return cf;
     }
 
+    @Override
+    public int getMaxDeleteObjectsNumber() {
```

Comment:
We don't need getMaxDeleteObjectsNumber anymore, the ObjectStorage implementation will handle the batch.

Reply:
this may be exposed by the caller to do the better batch.

like StreamObjectCompactor.compactCompositeObject

Reply:
StreamObjectCompactor may complete a compact in a few seconds。 It's not really necessary to remove complexity exposed to the outside just to gain a few seconds in advance.

Reply:
understand, i will change it.